CMakeLists.txt

```diff
@@ -10,34 +10,62 @@ project (
 if (EXISTS ${CMAKE_CURRENT_LIST_DIR}/ESMA_cmake)
   list (APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/ESMA_cmake")
   include (esma)
-else ()
-  if (NOT COMMAND esma) # build as standalone project
-
-    if (NOT SKIP_MEPO)
-       set (MEPO_INIT_COMMAND mepo init)
-       execute_process (
-          COMMAND ${MEPO_INIT_COMMAND}
-          WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-          )
-
-       set (MEPO_CLONE_COMMAND mepo clone)
-       execute_process (
-          COMMAND ${MEPO_CLONE_COMMAND}
-          WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-          )
-    endif()
-    option (SKIP_MEPO "Set to skip mepo steps" ON)
+endif ()
 
+# build as standalone project
+if (NOT COMMAND esma)
+  include (esma OPTIONAL)
+endif ()
+
+if (NOT COMMAND esma)
+
+  if (SKIP_MEPO)
+    message (FATAL_ERROR "ESMA not found")
+  else ()
+     set (MEPO_INIT_COMMAND mepo init)
+     execute_process (
+        COMMAND ${MEPO_INIT_COMMAND}
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+        )
+
+     set (MEPO_CLONE_COMMAND mepo clone)
+     execute_process (
+        COMMAND ${MEPO_CLONE_COMMAND}
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+        )
     list (APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/ESMA_cmake")
     include (esma)
-  endif()
-endif()
+    option (SKIP_MEPO "Set to skip mepo steps" ON)
+  endif ()
+
+endif ()
 
 option(BUILD_WITH_FLAP "Use FLAP for command line processing" ON)
 
 ecbuild_declare_project()
 
-find_package(PFLOGGER REQUIRED)
+if (NOT Baselibs_FOUND)
+  find_package(NetCDF REQUIRED Fortran)
+  add_definitions(-DHAS_NETCDF4)
+  add_definitions(-DHAS_NETCDF3)
+  add_definitions(-DNETCDF_NEED_NF_MPIIO)
+  add_definitions(-DHAS_NETCDF3)
+
+  find_package(HDF5 REQUIRED Fortran)
+  if(HDF5_IS_PARALLEL)
+     add_definitions(-DH5_HAVE_PARALLEL)
+  endif()
+
+  find_package(ESMF MODULE REQUIRED)
+
+  find_package(FLAP REQUIRED)
```

Comment:
Again, this packages should not be required.  There is a CMake OPTION to skip FLAP.  UFS should not need it.   FLAP provides command line option parsing for GEOS.

Reply:
Agreed. Pushed a commit to look for FLAP only if `BUILD_WITH_FLAP` is `ON`.

Reply:
Implemented in fb7562f